composer.json

```diff
@@ -11,22 +11,25 @@
             "email": "dcramer@gmail.com"
         }
     ],
-    "require-dev": {
-        "friendsofphp/php-cs-fixer": "~2.1",
-        "phpunit/phpunit": "^4.8 || ^5.0",
-        "ext-openssl": "*",
-        "nokitakaze/testhttpserver": "^0.1.0",
-        "monolog/monolog": "~1.0"
-    },
     "require": {
-        "php": ">=5.5",
-        "ext-hash": "*",
+        "php": ">=5.6",
```

Comment:
@Jean85 yes of cours 🤦‍♂️  stupid of me.

I'm not sure HHVM is still needed because most packages I am using and some big frameworks are also dropping it from their testing since it brings more trouble than it's worth.

Right now it's already on allow failure looking at the latest build for this PR.